include/dlaf/communication/executor.h

```diff
@@ -12,7 +12,7 @@
 /// @file
 
 #include <hpx/include/threads.hpp>
-#include <hpx/runtime/threads/executors/pool_executor.hpp>
+#include <hpx/runtime_local/pool_executor.hpp>
```

Comment:
I'd recommend you just include `hpx/include/parallel_executors.hpp` instead. It will also eventually be changed but you'll get a warning when that happens.